File: dataset/output/OPENAI_gpt-4o/Task128_OPENAI_gpt-4o.java
import java.util.Random;

public class RandomNumberGenerator {
    public static void main(String[] args) {
        System.out.println("Java random number: " + generateRandomNumber());
    }

    public static int generateRandomNumber() {
        Random rand = new Random();
        return rand.nextInt(100) + 1;
    }
}

File: dataset/output/OPENAI_gpt-4o/Task128_OPENAI_gpt-4o.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int generateRandomNumber() {
    return rand() % 100 + 1;
}

int main() {
    srand(time(0));
    printf("C random number: %d
", generateRandomNumber());
    return 0;
}